atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransaction.java

```diff
@@ -252,6 +255,16 @@ public boolean hasNext() {
         return ret;
     }
 
+    @Override
+    public ListenableFuture<Map<Cell, byte[]>> getAsync(TableReference tableRef, Set<Cell> cells) {
+        return Futures.transform(super.getAsync(tableRef, cells),
```

Comment:
Same here, let's figure out if we should extract something here vs copy.

Reply:
PR #4304, resolves this issue.